blueprints/factories/dataplex-datascan-factory/sample-specs/table-1/data-quality-scan.yml

```diff
@@ -0,0 +1,120 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [Optional] The resource name for the datascan. If not provided, the file path will be used as the datascan name.
+# The name of the datascan is created as the relative file path including directory names from the `datascan_spec_folder` input variable, substituting forward slashes for hyphens.  
+# name: "datascan"
+
+# The data source for DataScan. The source can be either a Dataplex `entity` or a BigQuery `resource`. If present, this variable will override the variable provided in datascan_defaults.
+# If present, this variable will override the variable provided in datascan_defaults.
+data:
+  resource: "//bigquery.googleapis.com/projects/bigquery-public-data/datasets/austin_bikeshare/tables/bikeshare_stations"
+
+# [Optional] DataQualityScan related setting. Variable descriptions are provided in https://cloud.google.com/dataplex/docs/reference/rest/v1/DataQualitySpec.
+data_quality_spec:
+  # [Optional] The percentage of the records to be selected from the dataset for DataScan.
+  # If present, this variable will override the variable provided in datascan_defaults.
+  sampling_percent: 100
+
+  # [Optional]  A filter applied to all rows in a single DataScan job. The filter needs to be a valid SQL expression for a WHERE clause in BigQuery standard SQL syntax. Example: col1 >= 0 AND col2 < 10
+  # If present, this variable will override the variable provided in datascan_defaults.
+  row_filter: "station_id > 1000"
+
+  # DataQualityScan rule configurations. Documentation for the supported rule types and rule specifications can be found in https://cloud.example.com/dataplex/docs/reference/rest/v1/DataQualityRule.
+  # Can optionally reference templated rules provided in the variable datascan_rules_library_file_path. Any variables provided here will override the default values provided in the rules library file.
+  rules:
+    - column: address
+      dimension: VALIDITY
+      ignore_null: null
+      non_null_expectation: {}
+      threshold: 0.99
+    - column: council_district
+      dimension: VALIDITY
+      ignore_null: true
+      threshold: 0.9
+      range_expectation:
+        max_value: '10'
+        min_value: '1'
+        strict_max_enabled: false
+        strict_min_enabled: true
+    - column: council_district
```

Comment:
Optional: It would be nice to have an example rule using the override + rule_template behavior here.

In the sample spec, we could add a file for the defaults.yml as well,
and then in the README, have one (working) example which shows us using defaults + overrides + template in a single module.